libs/dal/src/repositories/member/member.repository.ts

```diff
@@ -13,193 +10,129 @@ export interface IAddMemberData {
   memberStatus: MemberStatusEnum;
 }
 
-type MemberQuery = FilterQuery<MemberDBModel> & EnforceOrgId;
+export class MemberRepository implements IMemberRepository {
```

Comment:
I think this would require changing every reference of `MemberRepository` throughout the code. 
Either way I will try to change how this works by removing the `@Inject` requirement therefore removing `@nestjs/common` dependency from `@novu/dal` and possibly move the affected repositories to `@novu/ee-dal` and create 2x `MemberRepository` with same name one in dal and one in ee-dal - will see how that would work.